src/index.tsx

```diff
@@ -10,9 +10,9 @@ const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement
 );
 root.render(
-  <React.StrictMode>
-    <Provider store={store}>
-      <App />
-    </Provider>
-  </React.StrictMode>
+  // <React.StrictMode>
```

Comment:
I had a bag in reducers: when fetching and storing courses and authors I spread the elements of the current state  followed by the elements of the action.payload array , so the data was double rendered . I thought it was because of strict mode but actually I shouldn't return the elements of the current state but only the elements from action.payload array. Now i fixed the bag and added Strict Mode. Thanks!